src/Inviqa/Worldpay/Api/Request/PaymentService/Submit/Authorisation/Order/ShopperBasic.php

```diff
@@ -5,7 +5,7 @@
 use Inviqa\Worldpay\Api\Request\PaymentService\Submit\Authorisation\Order\Shopper\ShopperEmailAddress;
 use Inviqa\Worldpay\Api\XmlNodeDefaults;
 
-class ShopperApplePay extends XmlNodeDefaults
+class ShopperBasic extends XmlNodeDefaults
```

Comment:
Why was ShopperApplePay renamed to ShopperBasic?
Why not use a separate ShopperGooglePay class?
Are they exactly the same?

Reply:
yeah, Apple and Google use he exact same shopper class, so didn't see any point in creating 3 shopper classes

Shopper (for Worldpay)
ShopperApple
ShopperGoogle (same as Apple)